.gitignore

```diff
@@ -4,6 +4,9 @@
 # ignore CAssetManager for debugging
 /framework/web/CAssetManager.php
 
+# vendor directory
```

Comment:
I add this because if you use composer to manage your plugins, you will have a `vendor` folder to ignore.

Reply:
vendor dir is third-party
https://github.com/LimeSurvey/LimeSurvey/blob/a1cd6d20c7a5bd22b90cb8c4c3680fa432e18fc5/composer.json#L34

Reply:
I delete this entry in the `.gitignore` file.